src/Psl/Arr/values.php

```diff
@@ -16,6 +18,10 @@
  * @psalm-return   list<T>
  *
  * @psalm-pure
+ *
+ * @deprecated since 1.2, use Vec\values instead.
+ *
+ * @see Vec\values()
```

Comment:
Is there a specific need for Vec\values or can we enforce vec to always be a `list` type instead of `array<int, ...>` ?

Reply:
the return type here is what matters, you would use Vec\values when you want to create a *Vec* from *values* of an iterable.

Reply:
same for Vec\keys where you want to create a Vec from the keys of an iterable.